packages/apptive_grid_core/lib/model/link/apptive_link_type.dart

```diff
@@ -5,6 +5,9 @@ enum ApptiveLinkType {
   /// A link to the ApptiveObject itself
   self,
 
+  /// Updates a [Space], [Grid], [Form], [GridRow] or [GridField]
+  patch,
```

Comment:
What exactly is the difference between this and `update`?

Reply:
Maybe @bocasti can chime in on this?

Reply:
In general, 
- we use update methods in order to "fully" update a given resource. The submitted body is expected to represent the complete new state of the resource
- we use patch method in order to "partially" update a given resource. The submitted body is expected to provide a subset of properties to be updates. The properties not present in the body will remain untouched

So that would be the overall rule and difference between the 2. I hope that we do not diverge too much from that in practice :)